libvtcapture/include/vtcapture/vtcapture_openapi.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
```

Comment:
According to the debug info they left, file name of this one might be `vtCaptureApi_c.h`

Reply:
Yes you're right, I've renamed it.